docs/_static/style.css

```diff
@@ -1,4 +1,4 @@
-@import url("theme.css");
+@import url("styles/furo.css");
```

Comment:
The yaml css's color for numbers is black. It should be something else:
![Screenshot_20240429_110949](https://github.com/Ouranosinc/xclim/assets/91751417/bab786c8-8b85-4c55-8862-81173d25660b)

Reply:
This (I think) is set via the `pygments` library. I'll see what can be done.

Reply:
For this, we can probably just choose a different dark theme. Examples: https://stylishthemes.github.io/Syntax-Themes/pygments/

Reply:
I would propose "material" (since it's great for everything).

Reply:
@SarahG-579462 I've opted for `sas` and `lightbulb` as the `pygments` styles for code. Both of these themes are rated as being high contrast.

Reply:
The theming for the lexer of the YAML block in the notebook simply needs to be adjusted:
```
    "highlighted_py = highlight(pydata, PythonLexer(), Terminal256Formatter(style=\"manni\"))\n",
    "highlighted_yaml = highlight(ymldata, YamlLexer(), Terminal256Formatter(style=\"manni\"))\n",
    "highlighted_json = highlight(jsondata, JsonLexer(), Terminal256Formatter(style=\"manni\"))"
```